### PR TITLE
installed nodemon, set up database sql

### PR DESCRIPTION
**Pull request by KenDang1**

### CODE REVIEW

**Review by ciaci0028 — APPROVED** [review]
It looks great, Ken! You rock!

File: package.json [hunk 1]
```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server/server.js"
+    "start": "nodemon server/server.js"
```
Comment (jamespetran) [inline package.json:8]: i like nodemon

File: database.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        "gender" VARCHAR (1) NOT NULL,
+        "age" INTEGER NOT NULL,
+        "ready_to_transfer" BOOLEAN NOT NULL,
+        "notes" VARCHAR (160)
+    );
+
+INSERT INTO "koalas"
+    ("name", "gender", "age", "ready_to_transfer", "notes")
+VALUES
+    ('Scotty', 'M', 4, true, 'Born in Guatemala'),
```
Comment (jamespetran) [inline database.sql:14]: cool data

**Review by jamespetran — APPROVED** [review]

**Review by cdraz — APPROVED** [review]
Great work Ken!

**Pull request merged**